src/hardware/gameblaster.cpp

```diff
@@ -133,21 +133,24 @@ bool GameBlaster::MaybeRenderFrame(AudioFrame &frame)
 	static int16_t *p_buf[]           = {&buf[0], &buf[1]};
 	static device_sound_interface::sound_stream stream;
 
+	int16_t leftAccum = 0;
+	int16_t rightAccum = 0;
```

Comment:
(minor) we try to use snake_case for local variables.

Reply:
We previously accumulated the two SAA's outputs into`float`s because they could add up beyond the 16-bit range (which we soft-limited back into the 16-bit buffer, but this was removed in commit: 01fe221ba002a824472c2eb19ee4a57b1e05b7b3)

So to prevent that from wrapping, we should use `int`'s.

The resampler takes in int's (and has its own soft-clip protection), so atleast we won't be wrapping.

Reply:
Done!